src/api/stats/matic/getGoldenbullLpApys.js

```diff
@@ -0,0 +1,33 @@
+const getMasterChefApys = require('./getMaticMasterChefApys');
+
+const MasterChefAbi = require('../../../abis/matic/GoldenbullMasterChef.json');
+const pools = require('../../../data/matic/goldenbullLpPools.json');
+const { quickClient } = require('../../../apollo/client');
+const { addressBook } = require('blockchain-addressbook');
+const {
+  polygon: {
+    platforms: { goldenbull },
+    tokens: { GBULL },
+  },
+} = addressBook;
+const { quickLiquidityProviderFee } = require('./getQuickLpApys');
+const {
+  getScientificNotationFromTokenDecimals,
+} = require('../../../utils/getScientificNotationFromTokenDecimals');
+
+const getGoldenbullLpApys = async () =>
+  await getMasterChefApys({
+    masterchef: goldenbull.masterchef,
+    masterchefAbi: MasterChefAbi,
+    tokenPerBlock: 'gBullPerBlock',
+    hasMultiplier: true,
+    pools: pools,
+    oracleId: GBULL.symbol,
+    oracle: 'tokens',
+    decimals: getScientificNotationFromTokenDecimals(GBULL.decimals),
```

Comment:
nit: is it possible to have some extension function like `GBULL.eDecimals`, such a long name for just adding `1e` 😄

Reply:
I can't quickly figure out how to maintain the const typing while adding the extension. I'll just shorten the name for now